FizzBuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,11 @@
+def fizzBuzz(num)
+  if num % 3 == 0 && num % 5 == 0
+    return "FizzBuzz"
+  elsif num % 3 == 0
+    return "Fizz"
+  elsif num % 5 == 0
+    return "Buzz"
+  else
+    return num
```

Comment:
Great! And actually, since there is no code to run after the if/else, you don't even need to use the `return` keyword` in here at all. :]